net/ftpproxy/+POST_DEINSTALL

```diff
@@ -0,0 +1,33 @@
+DIFF='--- filter.inc.ftpproxy 2016-09-21 16:38:53.947075272 +0200
```

Comment:
@fbrendel there is anchor plugin code on core.git master now: https://github.com/opnsense/core/commit/edc40978997efd49422da71169fc177e7b40d8f2

Should be trivial to add your anchors with this now. Example migration for miniupnpd is included.